pkgs/by-name/dr/drm_info/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  libdrm,
+  json_c,
+  pciutils,
+  meson,
+  ninja,
+  pkg-config,
+  scdoc,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "drm_info";
```

Comment:
```suggestion
  pname = "drm-info";
```

Reply:
Are you sure this is correct? Because the upstream project is called `drm_info` and so is the command itself.